css/admin.css

```diff
@@ -496,4 +472,159 @@ body #post-body-content p.wp-clearfix:nth-child(1){
+.oo-poststuff .custom-input-field label {
+	width: 50%;
+}
+
+.oo-poststuff .custom-input-field input[type="button"] {
+	width: auto !important;
+	float: right;
+}
+
+.oo-poststuff #sortableFieldsList .custom-input-special,
+.oo-poststuff .menu-item-settings div.multiselect > div.onoffice-multiselect-popup > label,
+.oo-poststuff .custom-language {
+	display: flex;
+}
+
+.oo-poststuff .custom-language label,
+.oo-poststuff .custom-language input {
+	width: 50%;
+}
+
+.oo-poststuff #viewrecordsfilter .custom-input-field input,
+.oo-poststuff #viewrecordssorting .custom-input-field label,
+.oo-poststuff #viewrecordssorting .custom-input-field select,
+.oo-poststuff #sortableFieldsList .custom-input-field .custom-label,
+.oo-poststuff #sortableFieldsList .custom-input-field input[type="text"],
+.oo-poststuff #sortableFieldsList .custom-input-field select {
+	width: 50%;
+}
+
+.oo-poststuff #viewrecordssorting .chosen-container-single,
+.oo-poststuff #viewrecordssorting .chosen-container-multi {
+	width: 50% !important;
+}
+
+.oo-poststuff #sortableFieldsList .custom-input-field input[type="checkbox"],
+.oo-poststuff #viewformspecific .custom-input-field input[type="checkbox"],
+.oo-poststuff .custom-input-field input[type="checkbox"] {
+	margin-left: auto;
+	width: 16px;
+	min-width: 16px;
+	max-width: 16px;
+}
+
+.oo-poststuff #viewlayoutdesign .custom-input-field input[type="radio"] {
+	width: auto;
+}
+
+.oo-poststuff #viewrecordsfilter .custom-input-field label {
+	float: none;
+	width: 50%;
+}
+
+.oo-poststuff .custom-message,
+.oo-poststuff #viewrecordsfilter .inside div {
+	text-align: left;
+}
+
+.oo-poststuff #viewdocumenttypes label,
+.oo-poststuff #viewdocumenttypes select {
+	width: 100%;
+	max-width: 230px;
+}
+
+.oo-poststuff .custom-template input[type="radio"] {
+	width: auto;
+}
+
+.oo-poststuff #viewrecordssorting .chosen-container-single .chosen-single {
+	border-color: #8c8f94;
+	box-shadow: none;
+	border-radius: 3px;
+	padding: 0 24px 0 8px;
+	min-height: 30px;
+}
+
+.oo-poststuff #sortableFieldsList .custom-input-from-to {
+	flex-wrap: wrap !important;
+}
+
+.oo-poststuff #sortableFieldsList .custom-input-from-to label,
+.oo-poststuff #sortableFieldsList .custom-input-from-to input[type="number"],
+.oo-poststuff #sortableFieldsList .custom-input-from-to input[type="text"] {
+	flex: 0 50% !important;
+	width: 100% !important;
+	margin: 0 0 10px !important;
+}
+
+.oo-poststuff .custom-input-field input[type="hidden"] {
+	display: none;
+}
+
+.oo-poststuff #listSettings {
+	box-sizing: border-box;
+}
+
+.oo-poststuff input[type=checkbox],
+.oo-poststuff input[type=radio] {
+	margin: 0;
+}
+
+.oo-poststuff .sortable-item .custom-input-field:has(div.multiselect) {
+	display: block !important;
+}
+
+.oo-poststuff .custom-input-field .onoffice-multiselect-popup label {
+	display: flex;
+	flex-direction: row-reverse;
+}
+
+.oo-poststuff .sortable-item .custom-language select,
+.oo-poststuff .sortable-item .custom-language label {
```

Comment:
I think this label is already styled in line 509. Or will it be overwrite by some other styling?